tests/framework_azurefunctions/test_utilization.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from newrelic.common.utilization import AzureFunctionUtilization
+
+
+def test_utilization(monkeypatch):
+    monkeypatch.setenv("REGION_NAME", "EastUS2")
```

Comment:
Turns out this is case sensitive when being passed into the application: it should be all lowercase


```suggestion
    monkeypatch.setenv("REGION_NAME", "eastus2")
```